.circleci/config.yml

```diff
@@ -528,34 +529,34 @@ jobs:
           # We're not sure why this happens but it is related to the fact that we restore node_moduels
           # from the cache.
           name: Reconnect build
-          command: cd magnify && yarn build
```

Comment:
I think you can use `workspace`, you can see how we use it here on richie https://github.com/openfun/richie/blob/bf47a2795c17133c6506da12f13a235a9d1e85ea/docker-compose.yml#L79